cmd/clean.go

```diff
@@ -11,11 +11,18 @@ import (
 	"github.com/elastic/elastic-package/internal/cleanup"
 )
 
+const cleanLongDescription = `Use clean command to clean resources used for building the package.
```

Comment:
Make the first sentence consistent with other long descriptions.

```suggestion
const cleanLongDescription = `Use this command to clean resources used for building the package.
```

Reply:
Fixed